injected/src/features/broker-protection/captcha-services/utils/callback.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extracts a callback function name from a captcha container element's attributes.
+ *
+ * @param {Object} options - The options object
+ * @param {HTMLElement | null} options.captchaContainerElement - The DOM element containing the captcha
+ * @param {string} options.callbackAttrName - The name of the attribute containing the callback function name
+ * @returns {string} The callback function name extracted from the captcha element's attributes
+ */
+export function getCallbackFromAttribute({ captchaContainerElement, callbackAttrName }) {
```

Comment:
nit: this function and `getSiteKeyFromAttribute` are pretty much identical, they both return an attribute value, we could generalize into 

```suggestion
export function getAttributeValue({ element, attrName }) {
```

but I don't feel strongly